code/game/objects/items/ego_weapons/non_abnormality/shi.dm

```diff
@@ -1,14 +1,44 @@
 //Shi has 2 different modes. Dash Attacks and Boundary of Death.
 //Shi Assassin (Current one being used right now) uses Boundary of death.
 
+/obj/item/ego_weapon/city/shi_knife
+	name = "shi association knife"
+	desc = "A blade that is used by Shi Section 2 assassins to go out with honour."
+	special = "Attack yourself with this weapon to instantly kill yourself."
+	icon_state = "shi_dagger"
+	force = 44
+	damtype = RED_DAMAGE
+	armortype = RED_DAMAGE
+	attack_verb_continuous = list("pokes", "jabs", "tears", "lacerates", "gores")
+	attack_verb_simple = list("poke", "jab", "tear", "lacerate", "gore")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 60,
+							PRUDENCE_ATTRIBUTE = 60,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 60
+							)
+	var/force_update = 44
 
-//Boundary of death uses
+/obj/item/ego_weapon/city/shi_knife/attack(mob/living/target, mob/living/carbon/human/user)
+	..()
+	force = force_update
+	if(!CanUseEgo(user))
+		return
+	if(target == user)
+		force = 999999
+		for(var/mob/M in GLOB.player_list)
+			to_chat(M, "<span class='userdanger'>[uppertext(user.real_name)] has gone out with honor. 灰から灰へ</span>")
+		new /obj/effect/temp_visual/BoD(get_turf(target))
+		force_update *= 1.2
```

Comment:
As written, this hits the user for 44 red damage, then if the user is the target sets the force to 999999. This means they could stab someone with that.

Reply:
Sorry, I put the ..() in the wrong place. Fixed.